src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -63,7 +63,7 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
             return this.InvokeService(async () =>
             {
                 IList<Models.ProcessIdentifier> processesIdentifiers = new List<Models.ProcessIdentifier>();
-                foreach (IProcessInfo p in await _diagnosticServices.GetProcessesAsync(HttpContext.RequestAborted))
+                foreach (IProcessInfo p in await _diagnosticServices.GetProcessesAsync(processFilterOptions: null, HttpContext.RequestAborted))
```

Comment:
I think we might want to add a "new" route such as `/processes/$default` that will give you the process information of the configured default process.

Reply:
Maybe leave this future change.

Reply:
Let's discuss in the other issue.